app/views/transferSummary.scala.html

```diff
@@ -0,0 +1,92 @@
+@import java.util.UUID
+@import viewsapi.FormFunctions._
+@import helper.{inputRadioGroup => _, _}
+
+@(consignmentId: UUID, transferSummaryForm: Form[TransferSummaryData])(implicit request: RequestHeader, messages: Messages)
+@main(Messages("transferSummary.header")) {
+@defining(play.core.PlayVersion.current) { version =>
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <a href="@routes.DashboardController.dashboard()" class="govuk-back-link">Back</a>
+            @progressIndicator(Messages("transferSummary.progress"))
+            <h1 class="govuk-heading-xl">@Messages("transferSummary.title")</h1>
+            <dl class="govuk-summary-list">
+                <div class="govuk-summary-list__row">
+                    <dt class="govuk-summary-list__key">
+                        @Messages("transferSummary.seriesReference")
+                    </dt>
+                    <dd class="govuk-summary-list__value summary-list__right-column">
+
```

Comment:
Assume the content for each of the fields is to be added in a separate PR?

Reply:
Yes, as this information needs to be retrieved from the database; I'm working on this task now.